<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>655</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>655</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sources/resource.qrc">
    <normaloff>:/house icon.png</normaloff>:/house icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>商业贷款额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_shangdaie"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>公积金贷款额：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_gongjijine"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>按揭年数：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>11</y>
        <width>531</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="comboBox_anjienianshu">
         <item>
          <property name="text">
           <string>25年（300期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24年（288期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23年（276期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22年（264期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21年（252期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20年（240期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19年（228期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18年（216期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17年（204期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16年（192期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15年（180期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14年（168期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13年（156期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12年（144期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11年（132期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10年（120期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9年（108期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8年（96期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7年（84期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6年（72期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5年（60期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4年（48期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3年（36期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2年（24期）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1年（12期）</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>商贷利率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_shangdaililv"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>公积金利率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_gongjijinlilv"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>还款方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_benjin">
        <property name="text">
         <string>等额本金</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_benxi">
        <property name="text">
         <string>等额本息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_10" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_jisuan">
        <property name="text">
         <string>开始计算</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_qingkong">
        <property name="text">
         <string>清空重填</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
